lib/extensions/string_iec.rb

```diff
@@ -0,0 +1,12 @@
+class String
+  IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+  # https://github.com/jason-riddle/deploying-with-kubernetes/wiki/Conversion-Units
+  def parse_iec_number
```

Comment:
Also, can you add more documentation above the method (since it's been made to be more generic, it would be nice to show examples of inputs/outputs).  Also, we may want to move this into the more_core_extensions gem, possibly.